dogfooding/lib/screens/call_screen.dart

```diff
@@ -124,6 +124,7 @@ class _CallScreenState extends State<CallScreen> {
 
   @override
   Widget build(BuildContext context) {
+    // ignore: deprecated_member_use
     return WillPopScope(
```

Comment:
Maybe we should get rid of this deprecation otherwise we will not be able to support new Flutter version (this comes from experience 🙂 )

Reply:
It's tricky because the suggested replacement PopScope doesn't offer the same functionality. Lets leave it for now :)